extra/completions/alacritty.bash

```diff
@@ -35,9 +35,6 @@ _alacritty()
             compopt -o filenames
             COMPREPLY=( $(compgen -f -- "${cur}") )
             return 0;;
-        --dimensions | -d | --class | --title | -t)
-            # Don't complete here
-            return 0;;
```

Comment:
You probably don't want to remove `--class`, `--title` and `-t` here.

Reply:
That's a good point. Whoops.